harness/determined/common/experimental/resource_pool.py

```diff
@@ -15,7 +15,7 @@ class ResourcePool:
     def __init__(
         self,
         session: api.Session,
-        name: str = "",
```

Comment:
While you're touching things up, could you please adjust the comment in [Workspace.list_pools](https://github.com/determined-ai/determined/blob/40b4341f129a21d195690e7edf032c329672377d/harness/determined/common/experimental/workspace.py#L61) to conform with the [style guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)?